patch/kconfig-exclusions

```diff
@@ -16,6 +16,7 @@ CONFIG_MTD_SPI_NOR_USE_4K_SECTORS
 CONFIG_X86_PAT
 CONFIG_MLXSW_PCI
 ###-> mellanox_amd64-start
+CONFIG_IMA
```

Comment:
This'll be overidden when new hw-mgmt is integrated. Don't add between the markers